safe/impact_functions/inundation/flood_polygon_population/parameter_definitions.py

```diff
@@ -45,7 +65,11 @@ def affected_field():
 
 
 def affected_value():
-    """Generator for parameter stating what values constitute 'affected'."""
+    """Generate affected value parameter
```

Comment:
Please ensure docstring first line is a complete sentance (terminated with a full stop). Applies to all docstrings above too.